table.py

```diff
@@ -0,0 +1,79 @@
+"""High-level abstraction for Tables of multiple OpenFlow versions.
+"""
+import json
+from abc import ABC
+
+
+class TableStats(ABC):
```

Comment:
Do we really an `ABC` here? The intention is good, but since we're not using `@abstractmethod` I don't it helps much at the moment. We can keep evolving or subclassing this as needed in the future if ever needed. So, can you remove this inheritance for now?

Reply:
I agree.